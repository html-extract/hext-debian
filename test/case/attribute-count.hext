<div:attribute-count(3) @inner-html:attr_count/>
