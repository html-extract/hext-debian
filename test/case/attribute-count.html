<html>
  <head>
  </head>
  <body>
    <div>0</div>
    <div attr1="">1</div>
    <div attr1="" attr2="">2</div>
    <div attr1="" attr2="" attr3="">3</div>
  </body>
</html>
