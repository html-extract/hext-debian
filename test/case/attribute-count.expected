{"attr_count":"3"}
